src/extensions/monitoring.js

```diff
@@ -354,21 +294,17 @@ const Backlog = ({
     />
   </>
 }
-Backlog.displayName = 'Backlog'
-Backlog.propTypes = {
+BacklogTab.displayName = 'BacklogTab'
```

Comment:
just to be sure: this is not the caption actually displayed in UI? if it were, I think the `Tab`-suffix might seem a bit odd

Reply:
right, I think I raised exactly the same question before - will try to remember 🤞🏻